Update/advanced display rules (#832)
MapsPeople/docs: src/content/map/map-styling/display-rules.md

1. **Template** - Controls the information the Label should contain. Only applies to the CMS, any change at runtime will overwrite the information set up by the Template.
    * Location Name - Only displays the name of the Location.
    * External ID - Only displays the External ID of the Location.
    * External ID & Location Name - Displays both the External ID and the Location Name, with the External ID first.
    * Location Name & External ID - Displays both the Location Name and the External ID, with the Location Name first.
1. **Max width** - Specify how wide (in pixels) a Label can be before forcing a line-break.
    * A value of `0` will ensure no line-breaks for this label.

An example of in-app Display Rules using only "Label" Display Rules could look like this:

```json
{
    "displaySetting": {
        "labelVisible": true,
        "label": "{{name}}",
        "labelZoomFrom": 16.0,
        "labelZoomTo": 22.0,
        "labelMaxWidth": 0,
    }
}
```

![display-rules-polygon](/assets/map/display-rules/Display_Rules_Polygon.png)

> Polygon's operate separately from tiles. Tiles are drawn by MapsPeople and overlaid onto the mapping provider. Polygons are an overlay with customisable attributes that are then overlaid on top of the Tiles. Therefore, while you can edit the "Polygon" attributes of a Location connected to a Tile, be that an Area or a Room, you need to account for attributes such as the opacity of the Polygon in the resulting appearance. It is not currently possible to override the appearance of a Tile.

Reviewer: ```suggestion
Polygons are independent from tiles. Tiles are drawn by MapsPeople and overlaid onto the mapping provider. Polygons are an overlay with customisable attributes that are then overlaid on top of the Tiles. Therefore, while you can edit the "Polygon" attributes of a Location connected to a Tile, be that an Area or a Room, you need to account for attributes such as the opacity of the Polygon in the resulting appearance. It is not currently possible to override the appearance of Tiles.
```

Reviewer: Good addition here!

1. **Visibility** - Controls whether the Polygon is visible on the map.
    * The system will accept a Boolean here, so either `true` or `false`.
1. **Zoom from** - Sets the minimum Zoom Level at which the Polygon is visible.
    * The value should be a number between 1 and 22, with 1 being very far away, and 22 being very close. In a general use case, most users will only need values between 15 and 22.
    * If you are developing using the JavaScript SDK for Google Maps, the value must be an integer. If you are developing for Android or iOS, or using a different map provider, the value may be fractional.
1. **Zoom to** - Sets the maximum Zoom Level at which the Polygon is visible.
    * The value should be a number between 1 and 22, with 1 being very far away, and 22 being very close. In a general use case, most users will only need values between 15 and 22.
    * If you are developing using the JavaScript SDK for Google Maps, the value must be an integer. If you are developing for Android or iOS, or using a different map provider, the value may be fractional.
1. **Stroke color** - Controls the stroke color of the Polygon.

>     * You can select a colour using either a colour picker or entering an RGB (eg. 48, 113, 217) or HSL (eg. 217, 69%, 52%) value, or a HEX code (eg. #3071D9).

Reviewer: ```suggestion
    * In the CMS, you can select a color using the color picker displayed when clicking the color input field.
```

We try to use the American spelling of "color" and related terms throughout.

Reviewer: Please change this sentence in other applicable places.

>     * If setting the color in-app, the value provided must be in HEX code (eg. #3071D9).

Reviewer: ```suggestion
    * If setting the color in-app or through the Integration API, the value provided must be in HEX code (eg. #3071D9).
```

Are there other places we should also tell that you can edit it through the Integration API?
